src/OpenTK.NT/Native/User32/Enums/WindowMessage.cs

```diff
@@ -0,0 +1,298 @@
+﻿using UINT = System.UInt32;
+
+namespace OpenTK.NT.Native
+{
+    public enum WindowMessage : UINT
+    {
+        Null = 0x0000,
+        Create = 0x0001,
+        Destroy = 0x0002,
+        Move = 0x0003,
+        Size = 0x0005,
+        Activate = 0x0006,
+        SetFocus = 0x0007,
+        KillFocus = 0x0008,
+
+        //              public const uint SETVISIBLE           = 0x0009;
```

Comment:
What's up with this?

Reply:
No clue - this was already like this in the old API.cs, and I forgot to take a note to change it.
I'll just uncomment these, there shouldn't be any reason to not have these in the enum.